infrastructure/influxdb.conf

```diff
@@ -7,6 +7,8 @@
 #
 # Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
 
+
+bind-address = ":8088"
```

Comment:
@rikukissa can you explain why we need this and the 8086 port too please?

Reply:
Port 8086 is for InfluxDB clients to connect to the database, 8088 is a metadata port for backup / restore tool to connect to

Reply:
Understood.  Very cool!